### ISSUE TITLE
installing rancher on Rocky Linux

### ISSUE THREAD

**Issue by fahadshery** [rancher/rancher#37958]
**Rancher Server Setup**
- Rancher version: latest
- Installation option (Docker install/Helm Chart): Docker

System Info:
```bash
cat /etc/os-release
NAME="Rocky Linux"
VERSION="8.6 (Green Obsidian)"
ID="rocky"
ID_LIKE="rhel centos fedora"
VERSION_ID="8.6"
PLATFORM_ID="platform:el8"
PRETTY_NAME="Rocky Linux 8.6 (Green Obsidian)"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:rocky:rocky:8:GA"
HOME_URL="https://rockylinux.org/"
BUG_REPORT_URL="https://bugs.rockylinux.org/"
ROCKY_SUPPORT_PRODUCT="Rocky Linux"
ROCKY_SUPPORT_PRODUCT_VERSION="8"
REDHAT_SUPPORT_PRODUCT="Rocky Linux"
REDHAT_SUPPORT_PRODUCT_VERSION="8"
```
Docker version:
```
docker version
Client: Docker Engine - Community
 Version:           20.10.17
 API version:       1.41
 Go version:        go1.17.11
 Git commit:        100c701
 Built:             Mon Jun  6 23:03:11 2022
 OS/Arch:           linux/amd64
 Context:           default
 Experimental:      true

Server: Docker Engine - Community
 Engine:
  Version:          20.10.17
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.17.11
  Git commit:       a89b842
  Built:            Mon Jun  6 23:01:29 2022
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.6.6
  GitCommit:        10c12954828e7c7c9b6e0ea9b0c02b01407d3ae1
 runc:
  Version:          1.1.2
  GitCommit:        v1.1.2-0-ga916309
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```

**User Information**
- What is the role of the user logged in? Admin

**Describe the bug**
Can't install rancher using Docker. 

**To Reproduce**
1. Install Rocky Linux. 
2. Install Docker using the official docs and can install `sudo dnf install docker-ce docker-ce-cli containerd.io docker-compose-plugin --allowerasing --nobest`3. 
3. create rancher container: ` docker run -d --restart=unless-stopped \   -p 17008:80 -p 44643:443 \   -v /opt/rancher:/var/lib/rancher \   --privileged \ --name=rancher_docker_server \ rancher/rancher:latest`

**Result**
It just doesn't start at all and some of the logs are:
```
docker logs --tail 10 -f 3140cb92e9e2
2022/06/09 21:07:34 [INFO] Waiting for server to become available: Get "https://127.0.0.1:6444/version?timeout=15m0s": dial tcp 127.0.0.1:6444: connect: connection refused
2022/06/09 21:07:36 [INFO] Waiting for server to become available: Get "https://127.0.0.1:6444/version?timeout=15m0s": dial tcp 127.0.0.1:6444: connect: connection refused
2022/06/09 21:07:38 [INFO] Waiting for server to become available: Get "https://127.0.0.1:6444/version?timeout=15m0s": dial tcp 127.0.0.1:6444: connect: connection refused
2022/06/09 21:07:40 [INFO] Waiting for server to become available: Get "https://127.0.0.1:6444/version?timeout=15m0s": dial tcp 127.0.0.1:6444: connect: connection refused
2022/06/09 21:07:42 [INFO] Waiting for server to become available: Get "https://127.0.0.1:6444/version?timeout=15m0s": dial tcp 127.0.0.1:6444: connect: connection refused
2022/06/09 21:07:44 [INFO] Waiting for server to become available: Get "https://127.0.0.1:6444/version?timeout=15m0s": dial tcp 127.0.0.1:6444: connect: connection refused
2022/06/09 21:07:46 [INFO] Waiting for server to become available: Get "https://127.0.0.1:6444/version?timeout=15m0s": dial tcp 127.0.0.1:6444: connect: connection refused
2022/06/09 21:07:48 [INFO] Waiting for server to become available: Get "https://127.0.0.1:6444/version?timeout=15m0s": dial tcp 127.0.0.1:6444: connect: connection refused
2022/06/09 21:07:57 [FATAL] k3s exited with: exit status 2
INFO: Running k3s server --cluster-init --cluster-reset
```

I had the similar issue on `Ubuntu 22.04 LTS`...

## Netstat output
```
netstat -l -n -v -p -t
(Not all processes could be identified, non-owned process info
 will not be shown, you would have to be root to see it all.)
Active Internet connections (only servers)
Proto Recv-Q Send-Q Local Address           Foreign Address         State       PID/Program name
tcp        0      0 0.0.0.0:22              0.0.0.0:*               LISTEN      -
tcp        0      0 0.0.0.0:44643           0.0.0.0:*               LISTEN      -
tcp        0      0 0.0.0.0:32043           0.0.0.0:*               LISTEN      -
tcp        0      0 0.0.0.0:111             0.0.0.0:*               LISTEN      -
tcp        0      0 0.0.0.0:17008           0.0.0.0:*               LISTEN      -
tcp6       0      0 :::22                   :::*                    LISTEN      -
tcp6       0      0 :::44643                :::*                    LISTEN      -
tcp6       0      0 :::32043                :::*                    LISTEN      -
tcp6       0      0 :::111                  :::*                    LISTEN      -
tcp6       0      0 :::17008                :::*                    LISTEN      -
```
any solutions???

**Comment by LitchiInn** [comment]
Same for me. Possibly this is related to issue #36047

**Comment by fahadshery** [comment]
> Same for me. Possibly this is related to issue #36047

Have you fixed it?

**Comment by lcg72** [comment]
lsmod | grep ip_tables
if none, then 
modprobe ip_tables
modprobe ip_conntrack
modprobe iptable_filter
modprobe ipt_state

**Comment by hrvolapeter** [comment]
@lcg72 Thanks that worked

**Comment by Zermond** [comment]
I have too this issue with debian 11

**Comment by fahadshery** [comment]
still doesn't work for me...

> lsmod | grep ip_tables
> if none, then
> modprobe ip_tables
> modprobe ip_conntrack
> modprobe iptable_filter
> modprobe ipt_state

**Comment by dimitribellini** [comment]
> lsmod | grep ip_tables if none, then modprobe ip_tables modprobe ip_conntrack modprobe iptable_filter modprobe ipt_state

This message fix my problem to run Rancher 2.6.9 on my RockyLinux 8.5!!!

Thanks so much

**Comment by DwayneGit** [comment]
I found my problem was just low specs.
I was running it on a 1GB Ram and 1Core CPU virtual machine. 
beefed that up a little to 4GB Ram and 2Cores and all good
